src/samtools/samtools_fasta/config.vsh.yaml

```
@@ -0,0 +1,189 @@
+        required: true
+        direction: output
+  - name: Options
+    arguments:
+      - name: --no_suffix
+        alternatives: -n
+        type: boolean_true
+        description: |
+          By default, either '/1' or '/2' is added to the end of read names where the corresponding 
+          READ1 or READ2 FLAG bit is set. Using -n causes read names to be left as they are.
+      - name: --suffix
+        alternatives: -N
+        type: boolean_true
+        description: |
+          Always add either '/1' or '/2' to the end of read names even when put into different files.
+      - name: --use_oq
+        alternatives: -O
+        type: boolean_true
+        description: |
+          Use quality values from OQ tags in preference to standard quality string if available.
+      - name: --singleton
+        alternatives: -s
+        type: file
+        description: write singleton reads to FILE.
+      - name: --copy_tags
+        alternatives: -t
+        type: boolean_true
+        description: |
+          Copy RG, BC and QT tags to the FASTA header line, if they exist.
+      - name: --copy_tags_list
+        alternatives: -T
+        type: string
+        description: |
+          Specify a comma-separated list of tags to copy to the FASTA header line, if they exist. 
+          TAGLIST can be blank or * to indicate all tags should be copied to the output. If using *, 
+          be careful to quote it to avoid unwanted shell expansion.
+      - name: --read1
+        alternatives: -1
+        type: file
+        description: |
+          Write reads with the READ1 FLAG set (and READ2 not set) to FILE instead of outputting them. 
+          If the -s option is used, only paired reads will be written to this file.
+        direction: output
+      - name: --read2
+        alternatives: -2
+        type: file
+        description: |
+          Write reads with the READ2 FLAG set (and READ1 not set) to FILE instead of outputting them. 
+          If the -s option is used, only paired reads will be written to this file.
+        direction: output
+      - name: --output_reads
+        alternatives: -o
+        type: file
+        description: |
+          Write reads with either READ1 FLAG or READ2 flag set to FILE instead of outputting them to stdout. 
+          This is equivalent to -1 FILE -2 FILE.
+        direction: output
+      - name: --output_reads_both
+        alternatives: -0
+        type: file
+        description: |
+          Write reads where the READ1 and READ2 FLAG bits set are either both set or both unset to FILE 
+          instead of outputting them.
+        direction: output
+      - name: --filter_flags
+        alternatives: -f
+        type: integer
+        description: |
+          Only output alignments with all bits set in INT present in the FLAG field. INT can be specified 
+          in hex by beginning with `0x' (i.e. /^0x[0-9A-F]+/) or in octal by beginning with `0' 
+          (i.e. /^0[0-7]+/). [0]
```

Can we make these defaults more explicit?

```suggestion
          (i.e. /^0[0-7]+/). Default: `0`.
```